src/datachannel.cc

```diff
@@ -229,37 +229,39 @@ NAN_METHOD(DataChannel::Send) {
 
   DataChannel* self = Nan::ObjectWrap::Unwrap<DataChannel>(info.This());
 
-  if (info[0]->IsString()) {
-    Local<String> str = Local<String>::Cast(info[0]);
-    std::string data = *String::Utf8Value(str);
+  if (self->_jingleDataChannel != nullptr) {
```

Comment:
In a future PR, we should actually throw an InvalidStateError. See [here](https://w3c.github.io/webrtc-pc/#dfn-send()).

Reply:
https://github.com/js-platform/node-webrtc/issues/343